CHANGELOG.rst

```diff
@@ -55,6 +55,7 @@ Removed
 * Removed ``CentOS 6``/``RHEL 6`` support #4984
 
   Contributed by Amanda McGuinness (@amanda11 Ammeon Solutions)
+* Removed our fork of ``codecov-python`` and have switched back to the upstream version
```

Comment:
Would be good to reference the actual PR #
```suggestion
* Removed our fork of ``codecov-python`` for CI and have switched back to the upstream version (improvement) #5002
```

Not sure though if this enhancement really fits the `Removed` Changelog section as that's not the primary category.
Eg it's not a deprecation of the previously supported OS or st2.conf setting removal or change in some core st2 functionality we need to highlight in the `Removed` section.